patches/tModLoader/Terraria/Social/Steam/WorkshopHelper.TML.cs

```diff
@@ -718,7 +719,9 @@ internal class AQueryInstance
 					ReleaseWorkshopQuery();
 				}
 
-				internal SteamUGCDetails_t FastQueryItem(ulong publishedId) {
+				// if dependencyCount > 0 then query, otherwise ignore
+				// default calls will ignore dependencies
+				internal SteamUGCDetails_t FastQueryItem(ulong publishedId, uint dependencyCount = 0) {
```

Comment:
Making "get dependencies" a bool and using the pDetails from the same query to get # of deps it way better